packages/vit-s/src/shared-mui/container.js

```diff
@@ -27,6 +27,7 @@ export const useVitessceContainerStyles = makeStyles(theme => ({
       transition: 'height 200ms ease',
       // Custom
       height: 'auto',
+      marginTop: '30px',
```

Comment:
I did not find a better way to make the react-grid-layout not render on top of the "Expand/Collapse" button. It is because of the usage of `position: absolute` in `expandButtonContainer`, which is necessary for the button to be rendered on the right side of the screen